server/gfmd/user.c

```diff
@@ -1871,20 +1871,21 @@ gfm_server_user_auth_get(
 		return (GFARM_ERR_NO_ERROR);
 	}
 
-	e = gfarm_auth_user_id_type_parse(auth_user_id_type_str,
-					  &auth_user_id_type);
-	if (e != GFARM_ERR_NO_ERROR) {
-		gflog_debug(GFARM_MSG_UNFIXED, "invalid auth_user_id_type");
-		return (e);
-	}
-
 	/* XXX FIXME too long giant lock */
 	giant_lock();
 
-	if ((e = rpc_name_with_tenant(peer, from_client,
+	tenant = name_with_tenant ? NULL : process_get_tenant(process);
```

Comment:
both the name_with_tenant variable and the process variable are not initialized here